docs/source/running-jobs/many-jobs.rst

```diff
@@ -344,3 +344,9 @@ To have more customized control over generation of job variants, you can also us
         retry_until_up=True,
       )
       job_idx += 1
+
+
+Thousands of Jobs
```

Comment:
"Best Practices for Scaling"? Otherwise, too similar to "[Scale Out to Many Jobs](https://docs.skypilot.co/en/managed-jobs-docs/running-jobs/many-jobs.html#scale-out-to-many-jobs)".

Reply:
sure
```suggestion
Best Practices for Scaling
```